public/js/resolve-breaches.js

```diff
@@ -32,7 +32,12 @@
 
   function sendBreachDetailAnalyticsPing(eventCategory, eventAction, eventLabel) {
     if (typeof(ga) !== "undefined") {
-      ga("send", "event", eventCategory, eventAction, eventLabel);
+      // Set view pings as nonInteraction:true to get accurate bounce rate
+      let options = {};
+      if (eventAction !== "Engage"){
```

Comment:
@lesleyjanenorton - do we have any "Click" event actions?

Reply:
Both of these events would get marked as interactive:
https://github.com/mozilla/blurts-server/blob/master/public/js/resolve-breaches.js#L104
https://github.com/mozilla/blurts-server/blob/master/public/js/resolve-breaches.js#L92

Reply:
Does this mean that other events like "Submit" will be marked as non-interactive?

Reply:
What about changing this conditional to look specifically for eventAction(s) that should be marked as `nonInteraction: true` instead of adding `nonInteraction: true` to any eventAction that isn't `"Engage"`?